src/contexts/__tests__/current-product.test.tsx

```diff
@@ -1,3 +1,4 @@
+import { ReactNode } from 'react'
```

Comment:
This file needed to be updated since `CurrentProductProvider` now expects a `ProductData` object. I filled in empty values for the properties associated with that interface since other those aren't the focus of the tests here.